collectors/0/samza_prod_1.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+import sys
+
+from collectors.lib.samza_metric_reporter import SamzaMetricReporter
+
+CONSUMER_GROUP_ID = "tcollector_samza_prod_1"
+
+# FIXME: Should we set this as an ENV?
+KAFKA_BOOTSTRAP_SERVERS = [
+    'kafkaprod-1:9092',
+    'kafkaprod-2:9092',
+    'kafkaprod-3:9092',
+    'kafkaprod-4:9092',
+    'kafkaprod-5:9092',
+    'kafkaprod-6:9092'
+]
+
+
+def main():
+    reporter = SamzaMetricReporter(CONSUMER_GROUP_ID, KAFKA_BOOTSTRAP_SERVERS, metric_classes=['org.apache.samza.storage.kv.KeyValueStoreMetrics'])
```

Comment:
nit: define `['org.apache.samza.storage.kv.KeyValueStoreMetrics']` in a separate constant variable and reference that here, may be